---
permalink: /
title: "I exist"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

# Me

Hi there, I am Dieter. I read a lot on single-cell ML, large language models, (mechanistic) interpretability, and more. I am a PhD student at the UMCU in Utrecht, The Netherlands, working on clinically-relevant foundation model creation/training. Specifically, we are interested in using the (implicit) knowledge we have lying around in big databases like the [Human Cell Atlas](https://www.humancellatlas.org/), [CellxGene](https://cellxgene.cziscience.com/), [Gene Expression Omnibus](https://www.ncbi.nlm.nih.gov/geo/), and [GTEx](https://www.gtexportal.org/home/) and in biological (gene-regulatory) knowledge graphs such as [PrimeKG](https://www.nature.com/articles/s41597-023-01960-3)
to improve predictive capabilities for small patient groups (where omics dimensionality is easily ~20.000 for gene expression, up to 450.000 or 850.000 for methylation arrays, or [30 million genome-wide methylation sites](https://www.nature.com/articles/s41586-022-05580-6)). This high-dimensionality low-sample size (HDLSS) scenario is a nightmare for classical ML approaches, which is where [self-supervised representation learning](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9770283) and (foundational) embedding models(https://github.com/apeterswu/Awesome-Bio-Foundation-Models/blob/main/README.md) come in. 
